Kitodo/src/main/java/de/sub/goobi/metadaten/Metadaten.java

```diff
@@ -3971,7 +2703,6 @@ public User getCurrentUser() {
      *            input String
      */
     public void setWikiField(String[] inString) {
-        this.wikiField = inString;
```

Comment:
You are removing this field from class but not its setter and possible getter methods to access it. Why or why did you not even remove the according getter and setter methods?

Reply:
The setter was also removed in another commit, since the value of wikiField was never used.